plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts

```diff
@@ -948,6 +948,37 @@ describe('pg', () => {
       });
     });
 
+    it('should not add sqlcommenter comment when addSqlCommenterCommentToQueries=true is specified with a prepared statement', async () => {
+      instrumentation.setConfig({
+        addSqlCommenterCommentToQueries: true,
+      });
+
+      const span = tracer.startSpan('test span');
+      await context.with(trace.setSpan(context.active(), span), async () => {
+        try {
+          const query = 'SELECT NOW()';
+          const resPromise = await client.query({
+            text: query,
+            name: 'prepared sqlcommenter',
+          });
+          assert.ok(resPromise);
+
+          const [span] = memoryExporter.getFinishedSpans();
+          const commentedQuery = addSqlCommenterComment(
+            trace.wrapSpanContext(span.spanContext()),
+            query
+          );
+
+          const executedQueries = getExecutedQueries();
+          assert.equal(executedQueries.length, 1);
+          assert.equal(executedQueries[0].text, query);
+          assert.notEqual(query, commentedQuery);
```

Comment:
Does this line assert something meaningful?

If I get it right, `query` is a static text `SELECT NOW()` and `commentedQuery` will contain the comment even with your fix since the PR does not change it.

Reply:
Yeah you're correct this doesn't assert anything useful, removing it.